### DISCUSSION TITLE
Add a new category for the NoCode noobs.

### DISCUSSION THREAD

**Discussion by bobbyjunealive** [FlowiseAI/Flowise#833] [Ideas]
It's very hard to leap into conversations that I can kinda feel what's being said but I'm just to new to know. So to keep people like me that are going to be here away from the experienced coders, make a 'dum' area. We don't need help. We just need a spot. TY

**Comment by chungyau97** [comment]
Hi @bobbyjunealive,

category in our discord community or github?
